app/page.tsx

```
@@ -1,65 +1,36 @@
-import Image from "next/image";
+"use client";
+import { List } from "@/components/List";
+import { Select } from "@/components/Select";
+import { AnnouncementsReturn } from "@/db/getAnnouncements";
+import { useEffect, useState } from "react";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)